components/articles/Diary.js

```diff
@@ -144,20 +144,16 @@ class Diary extends Component {
 
 export const DiaryModel = {
   text: PropTypes.string.isRequired,
-  date: PropTypes.instanceOf(Date).isRequired,
+  date: PropTypes.PropTypes.oneOfType([PropTypes.string, PropTypes.instanceOf(Date)]).isRequired,
```

Comment:
Why is the different format here?

Reply:
I guess, the initial value of date automatically becomes string before the real data comes. I don't know yet how to deal with it

Reply:
@tyndria sounds like bug, imho should be one format: `number` in case of timestamp, or `Date` object otherwise. please, fix it.

Reply:
@Drapegnik hmm, number is a good idea